.vib/minio/cypress/cypress/integration/minio_test_suite.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.get('[data-testid="ErrorOutlineIcon"]').should('not.exist');
+  cy.get("#logout").scrollIntoView().should('be.visible').click();
+})
+
+it('allows creating a bucket and file upload', () => {  
+  cy.login();
+  cy.visit('add-bucket');
+  cy.fixture('testdata').then((td) => {
+    cy.get('#bucket-name').should('be.visible').type(`${td.bucketTitle}.${random}`);
+  })
+  cy.contains('button[type="submit"]','Create Bucket').click();
+  cy.contains('#upload-main','Upload').should('be.visible').click();
+  cy.contains('span','Upload File').should('be.visible').click();
+  cy.get('div#object-list-wrapper > input[type="file"]').selectFile('cypress/fixtures/example.json',
+  {force:true});
```

Comment:
Are you verifying somehow that the bucket and/or file were correctly created?

Reply:
I've added 2 additional assertions here. Check lines 28 and 29, please.